db/schema.rb

```diff
@@ -14,7 +14,7 @@
 ActiveRecord::Schema.define(version: 20170721001700) do
 
   create_table "ahoy_events", force: :cascade do |t|
-    t.integer  "visit_id"
+    t.uuid     "visit_id",   limit: 16
```

Comment:
@namangupta01 it seems it comes from here: https://github.com/openSUSE/osem/pull/1229

@differentreality @eug3nix could be a migration missed?

@namangupta01 anyway it is unrelated to your changes, so please exclude it from your PR. :wink: